github/conftest.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+# Fixtures to fetch data for the various GitHub branch checks
+
+# TODO: add doctests
+# TODO: convert to logger output
```

Comment:
looks like this is completed in this file?

Reply:
removed